main.go

```diff
@@ -88,6 +89,7 @@ func main() {
 	flagSet.String("approval-prompt", "force", "OAuth approval_prompt")
 
 	flagSet.String("signature-key", "", "GAP-Signature request signature key (algorithm:secretkey)")
+	flagSet.Var(&upstreamCAs, "upstream-ca", "paths to CA roots for the Upstream (target) Server (may be given multiple times, defaults to system trust store).")
```

Comment:
probably should be `upstream-tls-ca`

Reply:
just being consistent, the other two are: openshift-ca and 'tls-client'-ca, adding -tls- in there doesn't really add much value imo (to be pedant it would have to be -x509-ca :-)

Reply:
OpenShift CA should probably be openshift-tls-ca.  I can't change that now, but I think we should be consistent.  CA isn't unique enough, and we're already calling out tls-cert and tls-key.